### PR TITLE
Fixes #21959 - removed positional args from tests

### PR DESCRIPTION
**Pull request by lzap**
This fixes tests, thanks @orabin @orrabin not sure from my Vim editor :-)

### CODE REVIEW

File: test/functional/discovered_hosts_controller_test.rb [hunk 1]
```diff
@@ -34,23 +34,23 @@ def test_index_with_custom_column
...
     assert_select "td", /QEMU/
     assert_response :success
   end
 
   def test_show_multiple_actions
     host = discover_host_from_facts(@facts)
-    [:multiple_destroy, :select_multiple_organization, :select_multiple_location].each do |path|
-      xhr :post, path, {:host_ids => [host.id]}, set_session_user
+    [:multiple_destroy, :select_multiple_organization, :select_multiple_location].each do |action|
+      process action, method: :post, params: {:host_ids => [host.id]}, session: set_session_user
```
Comment (mmoll) [inline test/functional/discovered_hosts_controller_test.rb]: I guess this needs a `, xhr: true` added

Comment (lzap) [reply test/functional/discovered_hosts_controller_test.rb]: Added, tests passed without it, was not sure.

**Comment by lzap** [comment]
[test]

**Comment by orrabin** [comment]
@lzap I tested locally because the tests are broken without this. All the tests pass locally so I'm merging this.

**Comment by orrabin** [comment]
Thanks @lzap!

**Pull request merged**